sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -25,50 +28,9 @@ import { UserAgentOptions } from '@azure/core-http';
 import { WebSocketImpl } from 'rhea-promise';
 import { WebSocketOptions } from '@azure/core-amqp';
 
-// @public
-export interface AmqpAnnotatedMessage {
-    applicationProperties?: {
-        [key: string]: any;
-    };
-    body: any;
-    deliveryAnnotations?: {
-        [key: string]: any;
-    };
-    footer?: {
-        [key: string]: any;
-    };
-    header?: AmqpMessageHeader;
-    messageAnnotations?: {
-        [key: string]: any;
-    };
-    properties?: AmqpMessageProperties;
-}
+export { AmqpMessageHeader }
```

Comment:
Do we **have** to re-export AmqpMessageHeader, AmqpMessageProperties and the AmqpAnnotatedMessage from Service Bus? Assuming these will only be used by advanced users, is it too bad to expect them to import `@azure/core-amqp` in their files?

Reply:
Makes sense. Removed, I have.

Reply:
If we ever want, we can re-export